turbojpeg.py

```diff
@@ -227,6 +229,76 @@ def encode(self, img_array, quality=85, pixel_format=TJPF_BGR, jpeg_subsample=TJ
         finally:
             self.__destroy(handle)
 
+    """
+    @jpeg_buf           buffer containing jpeg image bytes
+    @pixel_format       pixel format of destination image
```

Comment:
Remove this.  We are not returning pixels from this function.  And we know we are dealing with YUV internally